test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -68,6 +70,12 @@ protected async Task<Octokit.PullRequest> WaitForPullRequestAsync(string targetR
 
             if (prs.Count == 1)
             {
+                // We use this method when we're creating the PR, and when we're fetching the updated PR
+                // We only want to set the Creation time when we're creating it
+                if (!_lastUpdatedPrTimes.ContainsKey(prs[0].Id))
```

Comment:
The variable name seems off? It says `lastUpdatedPrTimes` but we only store the creation time?

Reply:
Why not keep the latest time always? Should we have "else store the updateAt time"?

Reply:
We do store it, but only when it's different from the last time.
During Creation, we store the Creation time.
Afterwards, when we're doing an update, we call this same method to get the PR, but we don't want to update it again. We first want to check it in `WaitForUpdatedPullRequestAsync` bellow, and then, if they're different, update it

Reply:
If we had `"else store the updateAt time", then `_lastUpdatedPrTimes[pr.Id] != pr.UpdatedAt` would always return true, and we'd never return a PR